hyperdrive-cli/commands/stakewise/nodeset/upload-deposit-data.go

```diff
@@ -21,3 +21,19 @@ func uploadDepositData(c *cli.Context) error {
 	_, err = swcmdutils.UploadDepositData(sw)
 	return err
 }
+
+func registerNode(c *cli.Context) error {
+	// Get the client
+	hd, err := client.NewHyperdriveClientFromCtx(c)
+	if err != nil {
+		return err
+	}
+	sw, err := client.NewStakewiseClientFromCtx(c, hd)
+	if err != nil {
+		return err
+	}
+
+	_, err = sw.Api.Nodeset.RegisterNode(c.String("email"))
```

Comment:
See the comment on making the flag a global var above - you can reference the var's name instead of hardcoding the name here), [see here for an example](https://github.com/nodeset-org/hyperdrive/blob/b14e05283fd4371c2a58f56d01fa1173997475bb/hyperdrive-cli/commands/stakewise/wallet/generate-keys.go#L59).

Reply:
fixed